lib/pfconfig/namespaces/config/Pf.pm

```diff
@@ -191,6 +191,8 @@ sub build_child {
         $Config{'captive_portal'}{'secure_redirect'} = 'disabled';
         get_logger->info("secure redirect has been disabled since the portal certificate is a self-signed");
     }
+    $Config{'database_advanced'}{'net_read_timeout'} = normalize_time( $Config{'database_advanced'}{'net_read_timeout'} );
+    $Config{'database_advanced'}{'net_write_timeout'} = normalize_time( $Config{'database_advanced'}{'net_write_timeout'} );
```

Comment:
No need to do this.
Since it is already done because it is a time type in ```documentation.conf```

Reply:
I did it because pfconfig return the raw value if i am not doing that.

Reply:
Did you restart pfconfig and did a configreload after you updated documentation.conf